release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,54 @@
+# Test Infra Lead Playbook
+
+## Overview of Test Infra Lead responsibilities
+
+There are two major area that test-infra lead need to take care during the release cycle, which are:
+
+1. Create CI/Presubmit jobs for the new release, and polulate the testgrid dashboard
+2. Watch for test infra status, make sure test infra is stable, and react to test infra related issues
+
+It's important to ensure the stability of our test infra during the release cycle, so that we can get reliable testing signals throughout the release cycle.
+
+You can also work with @kubernetes/test-infra-maintainers or [test infra oncall](go.k8s.io/oncall) if you are blocked by anything.
```

Comment:
Add https:// before the link, otherwise it's a relative path.

Reply:
done